modules/cloud-function-v1/README.md

```diff
@@ -33,15 +33,15 @@ This deploys a Cloud Function with an HTTP endpoint, using a pre-existing GCS bu
 ```hcl
 module "cf-http" {
   source      = "./fabric/modules/cloud-function-v1"
-  project_id  = "my-project"
+  project_id  = var.project_id
   name        = "test-cf-http"
-  bucket_name = "test-cf-bundles"
+  bucket_name = var.bucket
   bundle_config = {
-    source_dir  = "fabric/assets/"
+    source_dir  = "fabric/tests/modules/cloud_function_v1/sample/"
```

Comment:
Adding remark from Roberto:

> Maybe we could provide better way (closer to the user needs) to use external assets / fixtures in the tests. 

Ideas for this:
1. create a symlink from `assets` to `tests/modules/<module_name>/assets`
2. create a symlink from `assets` to `tests/modules/assets`

Reply:
Done using 1 approach